cli/src/commands/parse/pst.rs

```diff
@@ -0,0 +1,1275 @@
+use anyhow::{anyhow, Context, Result};
+use chrono::{DateTime, TimeZone, Utc};
+use colored::Colorize;
+use itertools::Itertools;
+use libc::{c_char, c_int, size_t};
+use mailparse::{parse_header, MailHeader, MailHeaderMap};
+use reinfer_client::{
+    resources::{
+        attachments::AttachmentMetadata,
+        email::{EmailMetadata, Id as EmailId},
+    },
+    BucketIdentifier, Client, Mailbox, MimeContent, NewEmail,
+};
+use std::{
+    collections::HashSet,
+    path::{Path, PathBuf},
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc,
+    },
+};
+use structopt::StructOpt;
+
+use crate::progress::{Options as ProgressOptions, Progress};
+
+const ROOT_FOLDER_NAME: &str = "root";
+
+#[repr(C)]
+#[derive(Debug, Clone)]
+struct LibPffItem {
+    _unused: c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Clone)]
+struct LibPffRecordEntry {
+    _unused: c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Clone)]
+struct LibPffRecordSet {
+    _unused: c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Clone)]
+struct LibPffFile {
+    _unused: c_int,
+}
+
+#[repr(C)]
+#[derive(Debug, Clone)]
+struct LibcError {
```

Comment:
I took a look into the source code as I expected this to be `LibPffError` - I agree this is correct, I was wondering if you could use the `libc` crate here, but I don't know enough to comment

Basically, this is mildly surprising but looks fine